src/components/FeedbackSection.tsx

```diff
@@ -50,40 +36,30 @@ export interface FeedbackPropsType {
 }
 
 const Feedback = ({ result, document, input, secondAttempt }: FeedbackPropsType): JSX.Element | null => {
-  const Icon =
-    result === 'correct' || result === 'giveUp'
-      ? CorrectFeedbackIcon
-      : result === 'incorrect' || !secondAttempt
-      ? IncorrectFeedbackIcon
-      : AlmostCorrectFeedbackIcon
-
-  const background =
-    result === 'correct'
-      ? correct_background
-      : result === 'incorrect' || result === 'giveUp' || !secondAttempt
-      ? incorrect_background
-      : hint_background
-
-  const message =
-    result === 'correct'
-      ? 'Toll, weiter so! \nDeine Eingabe ist richtig.'
-      : result === 'incorrect' || !secondAttempt
-      ? `${result === 'incorrect' ? `Schade, deine Eingabe ist falsch.` : ``} Die richtige Antwort ist: ${
-          document?.article?.toLowerCase() === ARTICLES.diePlural ? 'die' : document?.article
-        } ${document?.word}`
-      : `Deine Eingabe ${input} ist fast richtig. Überprüfe Groß- und Kleinschreibung.`
+  let Icon, background, message
+  if (result === 'correct') {
+    Icon = CorrectFeedbackIcon
+    background = correct_background
+    message = 'Toll, weiter so! \nDeine Eingabe ist richtig.'
+  } else if (result === 'incorrect' || result === 'giveUp' || !secondAttempt) {
+    Icon = IncorrectFeedbackIcon
+    background = incorrect_background
+    message = `Die richtige Antwort ist: ${
+      document?.article?.toLowerCase() === ARTICLES.diePlural ? 'die' : document?.article
```

Comment:
I think you don't need those non null checks:
```suggestion
      document.article.toLowerCase() === ARTICLES.diePlural ? 'die' : document.article
```

Reply:
Ja document war in den Props optional, aber ich denke das ist nicht nötig / sinnvoll. Hab es mal unoptional gemacht.